javascript/javascript1/week3/warmup_exercise/script.js

```diff
@@ -0,0 +1,83 @@
+//ITEM ARRAY REMOVAL
+
+const names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+const nameToRemove = "Ahmad";
+
+for (let i = 0; i < names.length; i++) {
+  if (names[i] === nameToRemove) {
+    names.splice(i, 1);
```

Comment:
Very nice to consider that the name could be present multiple times! ⭐ As a small thought experiment, if you only were to remove one, can you come up with a good method on arrays that can help you get the index you need to splice? 😄

Reply:
Okay, thanks for guiding me in this. I have used indexOf method to find the index of first occuring element and then splice, which will remove only one "Ahmad" from the array, not multiple "Ahmad".